app/routes/regulatory-statements/edit.js

```diff
@@ -17,18 +17,18 @@ export default class RegulatoryStatementsEditRoute extends Route {
   }
 
   async model(params) {
-    const container = await this.store.findRecord(
+    const containerPromise = this.store.findRecord(
```

Comment:
What is the reason for this change?

Reply:
When adding a debugger statement here to look into what was going on, I realised that instead of passing promises to RSVP, which will run them all in parallel, we were instead running them in series using `await`. Rather than just put them back the way they were I thought I'd improve loading times a little by changing it over.

Reply:
Ok that makes sense!